src/components/SignUp/Training.tsx

```diff
@@ -16,6 +20,106 @@ const Training: React.FC<{
           <Tagline>Tell us where you learned to do what you do.</Tagline>
         </Col>
       </Row>
+      <Row>
+        <Col lg="4">
+          <Form>
+            <InputField
+              name="trainingInstitution"
+              onChange={setForm}
+              placeholder="Institution"
+              value={trainingInstitution || ''}
+            />
+          </Form>
+        </Col>
+      </Row>
+      <Row>
+        <Col lg="4">
+          <InputField
+            name="trainingCity"
+            onChange={setForm}
+            placeholder="City"
+            value={trainingCity || ''}
+          />
+        </Col>
+        <Col className="mt-4" lg="2">
+          <Form.Control as="select">
```

Comment:
@jameswilson425 this needs to update form data onchange also

Reply:
@bestfoodalex  added!